.jshintrc

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "./node_modules/finn-js-code-style/.jshintrc",
+    "browser": false,
+    "node": true,
+    "maxstatements": 10
```

Comment:
Is this `maxstatements` really needed? It doesnt seem to fail if I remove it locally..